hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/HoodieSyncConfig.java

```diff
@@ -94,28 +94,22 @@ public class HoodieSyncConfig extends HoodieConfig {
 
   public static final ConfigProperty<String> META_SYNC_PARTITION_EXTRACTOR_CLASS = ConfigProperty
       .key("hoodie.datasource.hive_sync.partition_extractor_class")
-      .defaultValue("org.apache.hudi.hive.MultiPartKeysValueExtractor")
+      .defaultValue("org.apache.hudi.hive.NonPartitionedExtractor")
```

Comment:
The default value of META_SYNC_PARTITION_FIELDS is "",  I think change the default value of META_SYNC_PARTITION_EXTRACTOR_CLASS to  NonPartitionedExtractor  is more semantic.

Reply:
@xicm the convention is we don't change default value unless there is a strong reason. we changed slash encoded extractor to multi-part extractor in 0.12.0 for handling usecases better. we don't want to change so often. besides, `MultiPartKeysValueExtractor` handles non partition case right?

Reply:
OK, thanks for your explanation.